### ISSUE TITLE
Force cache hits during automated testing

### ISSUE THREAD

**Issue by shawnmjones** [oduwsdl/MementoEmbed#44]
In spite of using a custom heuristic with `cachecontrol`, the system still skips the cache for some requests. Because web archives will likely block too many requests for the same URI-M, this will need to be alleviated for automated testing to be used with Travis CI.

**Comment by shawnmjones** [comment]
It does not appear that `cachecontrol` and `requests` actually work together.

```
import requests
import redis
from cachecontrol import CacheControl
from cachecontrol.caches.redis_cache import RedisCache

print("starting")

pool = redis.ConnectionPool(host='localhost', port=6379, db=0)
rconn = redis.Redis(connection_pool=pool)
session = CacheControl(requests.session(), RedisCache(rconn))

print(rconn.keys())
print("requesting...")
r = session.get("http://www.cs.odu.edu/~mln/")
print(rconn.keys())
print("done")
```

prints:

```
starting
[]
requesting...
[]
done
```

Nothing was written to the redis cache!

For RedisCache, the `cachecontrol` documentation notes "This is primarily a proof of concept, so please file bugs if there is a better method for utilizing redis as a cache."

I did not have any luck with FileCache either. Sometimes it would cache. Others it would not.

No caching headers on this URI either:
```
# curl -I "http://www.cs.odu.edu/~mln/"
HTTP/1.1 200 OK
Server: nginx
Date: Wed, 20 Jun 2018 21:36:32 GMT
Content-Type: text/html
Connection: keep-alive
Vary: Accept-Encoding
```

I may need to write my own caching system.

**Comment by shawnmjones** [comment]
This should be addressed by aec95d8
